mktags.bash

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+# Icinga 2 Docker image | (c) 2023 Icinga GmbH | GPLv2+
+set -exo pipefail
+
+if [[ "$1" =~ ^v((([0-9]+).([0-9]+)).[0-9]+)$ ]]; then
+  BUILDX=(docker buildx build --platform "$(cat "$(realpath "$(dirname "$0")")/platforms.txt")" --push)
+  cd "$(mktemp -d)"
+
+  cat <<<"FROM icinga/icinga2:${BASH_REMATCH[1]}" >Dockerfile
+
+  "${BUILDX[@]}" -t "icinga/icinga2:${BASH_REMATCH[2]}" .
```

Comment:
If I've parsed the use of `BASH_REMATCH` correctly, this will unconditionally update the `icinga/icinga2:2.x.y` tag? I'd also check if there's a newer version here, just to make this safe against rerunning the pipeline for an older release.

Reply:
We can re-run pipelines for older releases?

Reply:
Good question. I thought these workflows generally have a re-run button, but the one I checked didn't, but also I think they disappear after some time. So I can't say for sure.